src/main/java/uk/gov/hmcts/reform/hmc/api/utils/Constants.java

```diff
@@ -7,6 +7,15 @@ public final class Constants {
     public static final Boolean FALSE = false;
 
     public static final String LISTED = "LISTED";
+
+    public static final String COMPLETED = "COMPLETED";
+
+    public static final String ADJOURNED = "ADJOURNED";
+
+    public static final String PREPARE_FOR_HEARING_CONDUCT_HEARING = "Prepare for hearing";
+
+    public static final String DECISION_OUTCOME = "Hearing outcome";
+
```

Comment:
are these listing status or case status?

Reply:
It is not required anymore as we created a State Enum, so I have removed it.